src/Calculator/Views/GraphingCalculator/GraphingCalculator.xaml.cpp

```diff
@@ -197,12 +197,55 @@ void GraphingCalculator::OnEquationsVectorChanged(IObservableVector<EquationView
     GraphingControl->PlotGraph(false);
 }
 
+wstringstream GraphingCalculator::FormatTraceValue(double min, double max, float pointValue)
+{
+    wstringstream traceValueString;
+
+    // This formula will give us the precision to use for the pointValue based on the min/max. For example: 
+    // If we want to round to two decimal places, this will return 0.01
+    // If we want to round to the nearest 100th this will return 100
+    auto roundingNumber = pow(10, floor(log10(max - min)) - 3);
+
+    // Determine if we want to show scientific notation instead
+    if (roundingNumber <= 0.0000001 || roundingNumber >= 10000000)
```

Comment:
Why not rely on std::defaultfloat and use std::precision(min(-7, max(7, precision)) ?

Reply:
I don't know what the behavior is of std::defaultfloat but it wasn't respecting the precision that I passed to it.

I updated the code to use min and abs, it is unclear what a negative precision means so I avoided using that.